.circleci/config.yml

```diff
@@ -18,17 +28,21 @@ workflows:
           pod-install: true
       - ios/validate-podspec:
           name: Validate Podspec
-          xcode-version: "12.2.0"
-          podspec-path: WordPressAuthenticator.podspec
+          xcode-version: *xcode-version
+          podspec-path: *podspec
+          bundle-install: true
```

Comment:
This felt weird (vs using like `ios/install-dependencies`) but given that it's part of the published orb already, I don't see a lot of benefit to changing it now 🤔

Reply:
Yeah I didn't want to touch other parts of the orb, especially since changing things here would be breaking change that would require a bump of the orb to 2.x, which didn't feel necessary – in practice we try to avoid breaking changes in the orb and try our best to make retro-compatible changes unless absolutely necessary.